src/libcsg/modules/io/pdbreader.h

```diff
@@ -23,39 +23,45 @@
 #include <fstream>
 #include <votca/csg/topologyreader.h>
 #include <votca/csg/trajectoryreader.h>
+#include <votca/csg/atomtable.h>
 
 namespace votca { namespace csg {
 using namespace votca::tools;
 
 using namespace std;
 
 /**
-    \brief class for reading pdb files
+    brief class for reading pdb files
 
-    This class provides the TrajectoryReader + Topology reader interface
+    This class provides the Trajectory and Topology reader interface
     for pdb files
 
 */
 class PDBReader :
-    public TrajectoryReader, public TopologyReader
+    public TopologyReader , public TrajectoryReader, public AtomTable
 {
     public:
+
+        /// Constuctor
         PDBReader() {}
+        /// Destructor
         ~PDBReader() {}
-
         /// open a topology file
         bool ReadTopology(string file, Topology &top);
-
-        /// open a trejectory file
+        /// open a trajectory file
         bool Open(const string &file);
         /// read in the first frame
         bool FirstFrame(Topology &top);
         /// read in the next frame
         bool NextFrame(Topology &top);
+        /// set the topology file name: <name>.pdb (convention: ".pdb"="FIELD")
```

Comment:
Hmm, I think the `convention` part is wrong here and 2 lines below!

Reply:
What should it be?

Reply:
No convention!

Reply:
Ah, ok, I wasn't sure what the comment meant i the first case. I'll fix it.

Reply:
Wait for @MrTheodor's comments too.

Reply:
I'm guessing this is true for both the set and get functions?

Reply:
Yes!

Reply:
What is this ```convention``` ? I see it only in dlpolyreader/writer. And where those setters/getters are used?